DirectProgramming/C++SYCL_FPGA/Tutorials/Features/agilex5_dsp_mode/README.md

```diff
@@ -0,0 +1,265 @@
+      ```
+   4. Compile for FPGA hardware (longer compile time, targets FPGA device):
+      ```
+      nmake fpga
+      ```
+> **Note**: If you encounter any issues with long paths when compiling under Windows*, you may have to create your 'build' directory in a shorter path, for example c:\samples\build.  You can then run cmake from that directory, and provide cmake with the full path to your sample directory, for example:
+>
+>  ```
+  > C:\samples\build> cmake -G "NMake Makefiles" C:\long\path\to\code\sample\CMakeLists.txt
+>  ```
+### Read the Reports
+
+Locate `report.html` in the `Agilex5_DSP_report.prj/reports/` directory.
+
+On the main report, navigate to the top right tab *Area Estimates* page, and click to expand on the "Kernel System" row in the Area Estimates table. Identify the row "mult", and scroll right to see the DSP resource usage for the kernel's computation. You should be able to see that the DSP resource used is 1.
+
+Repeat the build step, but this time with Agilex 7 passed in as the device family,
+```   
+cmake .. -DFPGA_DEVICE=Agilex7
+```
+Locate `report.html` in the `Agilex7_DSP_report.prj/reports/` directory, navigate to the same Area estimates table, you should be able to see that the DSP resource used is now 2. 
+
+You can also check the directory `kernel_hdl/` under `Agilex5_DSP_report.prj/` where all the generated RTL is stored. Doing a quick grep for `"complex_mult"`, we can see that the `"operation_mode"` is set to `"complex_mult"` when compiling for Agilex5, as opposed to `"operation_mode("m18x18_sumof2)"` when we compile to Agilex7.
+
+## Run the `Agilex 5 DSP mode` Sample
+
+### On Linux
+
+1. Run the sample on the FPGA emulator (the kernel executes on the CPU).
+   ```
+   ./Agilex5_DSP.fpga_emu
+   ```
+2. Run the sample of the FPGA simulator device (the kernel executes on the CPU).
+   ```
+   CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1 ./Agilex5_DSP.fpga_sim
+   ```
+3. Run the sample on the FPGA device (only if you ran `cmake` with `-DFPGA_DEVICE=<board-support-package>:<board-variant>`).
+   ```
+   ./Agilex5_DSP.fpga
+   ```
+
+### On Windows
+
+1. Run the sample on the FPGA emulator (the kernel executes on the CPU).
+   ```
+   Agilex5_DSP.fpga_emu.exe
+   ```
+2. Run the sample of the FPGA simulator device (the kernel executes on the CPU).
+   ```
+   set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1
+   Agilex5_DSP.fpga_sim.exe
+   set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=
+   ```
+3. Run the sample on the FPGA device (only if you ran `cmake` with `-DFPGA_DEVICE=<board-support-package>:<board-variant>`).
+   ```
+   Agilex5_DSP.fpga.exe
+   ```
+
+## Example Output
+
+You will see the device used. If successful, the program displays output similar to the following:
+
+```
+PASSED: kernel results are correct.
+```
+
+
+### Understand the Results
+
+Selecting the Agilex 5 family for performing ac_complex multiplications can help lower the DSP resource usage by half in comparison to the Agilex 7 family.
```

Comment:
I'd say that this is not the conclusion.
The conclusion is that our compiler is able to take advantage of the hardware capabilities of Agilex5
I'm questioning the need for this sample.
Were you given a case to create this sample?
Because in the end, it's just using * with ac_complex.
I don't understand what we are trying to teach to the users here
This could be simplified as a note in the ac_complex sample: When targeting Agilex5, you'll get a complex mult using a single DSP